.github/workflows/integration-windows.yaml

```diff
@@ -0,0 +1,39 @@
+name: Cloud Hypervisor Tests (Wingows Guest)
+on: [pull_request, create]
+
+jobs:
+  build:
+    if: github.event_name == 'pull_request'
+    name: Tests (Windows Guest)
+    runs-on: garm-jammy
```

Comment:
Just a reminder that Windows guest tests are also tested on aarch64, with musl build only though.